src/rust/engine/process_execution/src/remote.rs

```diff
@@ -757,8 +757,8 @@ impl crate::CommandRunner for CommandRunner {
     // Ensure the action and command are stored locally.
     let (command_digest, action_digest) = with_workunit(
       context.workunit_store.clone(),
-      "ensure_action_stored_locally".to_owned(),
-      WorkunitMetadata::with_level(Level::Debug),
+      format!("ensure_action_stored_locally for {:?}", action),
```

Comment:
Is this okay to do?

Reply:
Rather than a name, this might be a description instead? Would want to set that on the `WorkunitMetadata`.

Cleanups to this API welcome!